proposals/0044-advisories.md

```diff
@@ -0,0 +1,63 @@
+- Supersede GitHub issues for bug reporting or monitoring.
+- Suggest backporting bug fixes to older releases.
+- Public blaming/shaming for faulty code that was released.
+
+# Detailed Design
+
+Advisories will be implemented as a separate content type (structured content collection) in Astro, with robust schema validation using Zod. Each advisory will be stored as a Markdown file with YAML frontmatter containing essential metadata: title, subtitle, category (pipelines, modules, subworkflows, configuration), type (known_regression, incompatibility, security, performance, data_corruption, scientific_advice, other), severity (low, medium, high, critical), publishedDate, reporter, and reviewer information.
+
+The website will display advisories through several new components and routes: AdvisoryHeader for metadata and categorization, AdvisoryCard for grid-based listing, and AdvisoryLayout for the overall page structure (note: AdvisoryLayout implementation is pending discussion). The UI will feature visual indicators using badges and icons, with distinct color schemes and icons for each advisory type and severity level. The system will support dynamic routing with type-specific pages and include a sidebar navigation for easy access to different advisory categories.
+
+The implementation will include basic filtering and search capabilities. Advisories published within the last three months will be automatically categorized as "current" and prominently displayed on the website.
+
+# Drawbacks
+
+- The YAML frontmatter structure is complex to accommodate diverse use cases and affected components, requiring authors to learn the available fields and which values are acceptable.
+- The criteria for when to create an advisory versus resolving an issue through Slack or the Seqera community needs to be clearly defined.
+
+# Alternatives
+
+- Maintain the current approach of communicating issues through Slack and blog posts, without implementing a formal advisory system.
+
+# Adoption Strategy
+
+Initially, we will allow any nf-core maintainer and potentially users to submit advisories through pull requests to the website. The pull request reviewers should add themselves as advisory reviewers in the frontmatter. This process can be formalized later?
+
+# Unresolved Questions
+
+- Technical implementation details need to be resolved, including:
+  - Integration with existing Astro layouts without creating a separate layout for this content type (Request by @mashehu)
+  - Implementation of search functionality with version range support (Request by @SPPearce)
+  - Integration into the pipeline pages for each affected revision (Subject to a separate pull request because of technical complexity?)
+  - Integration into the module/subworkflow pages (Subject to discussion, whether desirable at all.)
```

Comment:
```suggestion
- [] Technical implementation details need to be resolved, including:
  - [x] Integration with existing Astro layouts without creating a separate layout for this content type (Request by @mashehu)
  - [] Implementation of search functionality with version range support (Request by @SPPearce)
  - [] Integration into the pipeline pages for each affected revision (Subject to a separate pull request because of technical complexity?)
  - [] Integration into the module/subworkflow pages (Subject to discussion, whether desirable at all.)
```